packages/fxa-payments-server/src/routes/Subscriptions/index.test.tsx

```diff
@@ -318,6 +330,24 @@ describe('routes/Subscriptions', () => {
     await findByTestId('error-loading-customer');
   });
 
+  it('displays an error if subsequent invoice fetch fails', async () => {
```

Comment:
Could you please add an additional test for SubscriptionItem to test that it shows the SubsequentInvoice error dialog?

Something similar to `it('should display an error message for a plan found in auth-server but not subhub'`

Reply:
added